### PR TITLE
CSV dataset importing

### PR DESCRIPTION
**Pull request by gentlecat**

### CODE REVIEW

**Comment by gentlecat** [comment]
Live at http://test.acousticbrainz.org/datasets/import.

File: templates/datasets/import.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+{% block content %}
+  <div id="dataset-viewer">
+    <h2 class="page-title">Datasets</h2>
+    <h3>Import dataset</h3>
+
+    <div class="alert alert-info" role="alert">
+      <strong>You can import datasets in <a href="https://en.wikipedia.org/wiki/Comma-separated_values">CSV</a> format.</strong>
+      Each row in the file must contain information about a single class. First
+      column is a class name followed by MBIDs of recordings in that class.
```
Comment (alastair) [inline templates/datasets/import.html]: I think it'd be nicer to have 1 mbid per row, with `mbid,class`

File: templates/datasets/import.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
+{% extends 'base.html' %}
+
+{% block title %}Import dataset - AcousticBrainz{% endblock %}
+
+{% block content %}
```
Comment (alastair) [inline templates/datasets/import.html]: Can we have a back button?

Comment (gentlecat) [inline templates/datasets/import.html]: Back to profile page?

Comment (alastair) [inline templates/datasets/import.html]: yep

File: templates/datasets/import.html (hunk 1 above)
Comment (gentlecat) [inline templates/datasets/import.html]: Done. :sparkles:

**Comment by alastair** [comment]
:+1:

**Pull request merged**